packages/entity-server/src/models/types.ts

```diff
@@ -6,12 +6,49 @@
 import { DatabaseModel, DatabaseType } from '@tupaia/database';
 import { ObjectLikeKeys, Flatten } from '../types';
 
-type PartialOrArray<T> = {
-  [field in keyof T]?: T[field] | T[field][];
+type FilterComparators = '!=' | 'ilike';
```

Comment:
```suggestion
type FilterComparators = '!=' | 'ilike' | '=';
```
See comments about this in `filter.ts`